server/mysql_storage_provider.go

```diff
@@ -58,6 +59,7 @@ func newMySQLStorageProvider(mf monitoring.MetricFactory) (StorageProvider, erro
 		if *maxConns > 0 {
 			db.SetMaxOpenConns(*maxConns)
 		}
+		db.SetMaxIdleConns(*maxIdle)
```

Comment:
The go doc for this implies that the default value (currently 2) could change in future. This makes picking a default for the flag somewhat problematic.

I suggest making the default for the flag -1 and making this test `>= 0`. Then if it's not set and Go change their minds in future users will still get the new default.

Reply:
Thanks that's a great point. Updated to reflect your suggestion.

Reply:
Thanks.